iml-gui/crate/src/page/volumes.rs

```diff
@@ -34,6 +34,7 @@ type Row = (Arc<VolumeRecord>, Vec<Arc<VolumeNodeRecord>>, Vec<Arc<Host>>);
 
 #[derive(Default)]
 pub struct Model {
+    pub(crate) host: Option<Arc<Host>>,
```

Comment:
This could be a list of hosts, but I can't imagine a use case yet.

Reply:
I don't see this changing from a single host.